CHANGES.rst

```diff
@@ -4,6 +4,8 @@ ChangeLog
 +------------+---------------------------------------------------------------------+------------+
 | Version    | Description                                                         | Date       |
 +============+=====================================================================+============+
+| **2.0.0**  | * BREAKING CHANGE: Improved framebuffer performance                 | TBC        |
```

Comment:
I'm not a fan of starting with the "negative" and then pointing out a big positive. Adding `(breaking change)` lowercase at end maybe? Same for the title of this pull request.

Reply:
sure thing

Reply:
although, this 'breaking change' would not be visible to consumers of the displays, only between luma.core <=> luma.lcd, luma.oled

Reply:
yea, downplaying it isn't a problem, this looks better.